### PR TITLE
Remove target from the evaluator config.

### PR DESCRIPTION
**Pull request by devang-ml**
## Describe your changes
Remove target from the evaluator config.
## Checklist before requesting a review
- [ ] Add unit tests for this change.
- [ ] Make sure all tests can pass.
- [x] Update documents if necessary.
- [x] Format your code by running `pre-commit run --all-files`

### CODE REVIEW

File: docs/source/overview/quicktour.md [hunk 1]
```diff
@@ -115,7 +114,9 @@ The engine which handles the auto-tuning process. You can select search strategy
 
 ```json
 "engine": {
-    "cache_dir": ".cache"
+    "host": {"type": "LocalSystem"},
```
Comment (guotuofeng) [inline docs/source/overview/quicktour.md:117]: this line looks different with the example https://github.com/microsoft/Olive/blob/main/examples/resnet_ptq_cpu/resnet_config.json#L101

Do we support both of them?

File: docs/source/overview/quicktour.md [hunk 2]
```diff
@@ -115,7 +114,9 @@ The engine which handles the auto-tuning process. You can select search strategy
...
+    "target": {"type": "LocalSystem"},
```
Comment (guotuofeng) [inline docs/source/overview/quicktour.md:118]: >target [](http://example.com/codeflow?start=5&length=6)

it seems the engine need to be changed to host the target?

File: docs/source/overview/quicktour.md (hunk 1 above)
Comment (jambayk) [reply docs/source/overview/quicktour.md:117]: yes, `host` can refer to a name in the `systems` list or be a system config dict.

File: docs/source/overview/quicktour.md (hunk 2 above)
Comment (jambayk) [reply docs/source/overview/quicktour.md:118]: yes, I have opened a PR into this branch to add the other necessary changes. https://github.com/microsoft/Olive/pull/233 

@devang-ml please review, the linked PR and merge it into this branch.

**Review by xiaoyu-work — APPROVED** [review]

**Review by guotuofeng — APPROVED** [review]

**Pull request merged**